packages/@okta/vuepress-site/docs/guides/secure-oauth-between-orgs/main/index.md

```diff
@@ -0,0 +1,311 @@
+2. In the hub org, [create an OAuth 2.0 service app](#create-an-oauth-2-0-service-app-in-the-hub-org) with `private_key_jwt` authentication method and JWKS public key of each Org2Org app in the spoke orgs. Grant [allowed scopes](#request-examples) to the hub org service app (the OAuth 2.0 client).
+3. [Set and activate provisioning in the Org2Org apps](#enable-provisioning-in-the-org2org-app) from the Okta API.
+4. [Assign users and groups to be pushed to the hub org](#assign-users-and-groups-in-the-org2org-app).
+
+### Add an Org2Org app integration in a spoke org
+
+In the spoke org, where you want to push users and groups to the central hub org, you need to add an instance of the Org2Org app integration by using the [Okta Apps API](/docs/reference/api/apps/#add-basic-authentication-application). This generates an Org2Org app integration instance and the key certificates required to connect to the hub org.
+
+As an Okta admin, make a `POST /api/v1/apps` request to the spoke org with [Okta Org2Org parameters](/docs/reference/api/apps/#add-okta-org2org-application):
+
+| Parameter |  Description/Value   |
+| --------- |  ------------- |
+| `name`  |  `okta_org2org` |
+| `label`  |  Specify a label for this Org2Org app integration. |
+| `baseUrl`  |  Specify your hub org base URL. |
+| `signOnMode`  |  You can set this parameter to any valid value, but if you specify `SAML_2_0`, the Org2Org app signing certificate appears in the Admin Console. |
+
+##### Request example
+
+```bash
+curl -v -X POST \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+-d '{
+  "name": "okta_org2org",
+  "label": "My spoke Org2Org App",
+  "signOnMode": "SAML_2_0",
+  "settings": {
+    "app": {
+        "baseUrl": "https://${yourHubOktaDomain}"
+    }
+  }
+}' "https://${yourSpokeOktaDomain}/api/v1/apps"
+```
+
+From the response of your POST request, use the `id` property of the Org2Org app instance to [retrieve the key credentials](/docs/reference/api/apps/#list-key-credentials-for-application) generated for the app with the `GET /api/v1/apps/${id}/credentials/keys` request.
+
+##### Request example
+
+```bash
+curl -v -X GET \
+-H "Accept: application/json" \
+-H "Content-Type: application/json" \
+-H "Authorization: SSWS ${api_token}" \
+"https://${yourOktaDomain}/api/v1/apps/${yourOrg2OrgAppId}/credentials/keys"
+```
+
+##### Response example
+
+```json
+[
+  {
+    "created": "2015-12-10T18:55:35.000Z",
+    "expiresAt": "2045-01-23T02:15:23.000Z",
+    "x5c": [
+      "MIIDqDCCApCgAwIBAgIGAUsUko … llBooPPS3wYlNA=="
+    ],
+    "e": "AQAB",
+    "n": "htbi5H5MN_oYaKcZ … hVagjTzo",
+    "kty": "RSA",
+    "use": "sig",
+    "x5t#S256": "7CCyXWwKzH4P6PoBP91B1S_iIZVzuGffVnUXu-BTYQQ"
+  }
+]
+```
+
+> **Note**: The keys are truncated for brevity.
+
+Save the returned key credentials for use in the hub org service app configuration.
```

Comment:
Maybe:
Save the returned key credentials for use in the service app configuration for the hub org.
?